config/main.py

```diff
@@ -58,9 +59,11 @@
 CFG_LOOPBACK_NAME_TOTAL_LEN_MAX = 11
 CFG_LOOPBACK_ID_MAX_VAL = 999
 CFG_LOOPBACK_NO="<0-999>"
+PIPE_FILE = "/tmp/reload.txt"
```

Comment:
`PIPE_FILE` is not very descriptive for a global constant. If this needs to be global, it should at least be more descriptive, like `RELOAD_PIE_FILE` or similar.